etc/fetchers/__tests__/utils.test.ts

```diff
@@ -46,3 +48,47 @@ describe("utils > parseFAQ", () => {
     ]);
   });
 });
+
+describe("utils > extractGoogleQuery", () => {
+  it("should extract query from a single link element", () => {
+    const el = `<a href="https://www.google.com/url?q=http://wa.me/6281286109493&sa=D&source=editors&ust=1627902606786000&usg=AOvVaw0Ou4V_199oMgGoMYp8FaAW">
+        http://wa.me/6281286109493
+      </a>`;
+
+    const extracted = extractGoogleQuery(el);
+    expect(extracted).not.toBe(el);
+
+    const $ = cheerio.load(extracted);
+    const parsedEl = $("a");
+    expect(parsedEl.attr("href")).toBe(parsedEl.text().trim());
+  });
+
+  it("should extract query from multiple link elements", () => {
+    const el = `<div>
+      <a href="https://www.google.com/url?q=http://wa.me/6281286109493&sa=D&source=editors&ust=1627902606786000&usg=AOvVaw0Ou4V_199oMgGoMYp8FaAW">
+        http://wa.me/6281286109493
+      </a>
+      <a href="https://www.google.com/url?q=https://3r-medika-gas-isi-ulang-gas-oxygen-medis.business.site/?utm_source%3Dgmb%26utm_medium%3Dreferral&sa=D&source=editors&ust=1627902606786000&usg=AOvVaw3UsJ1uI79Io9KslmnQtDXq">
+      https://3r-medika-gas-isi-ulang-gas-oxygen-medis.business.site/?utm_source=gmb&amp;utm_medium=referral
+      </a>
+    </div>`;
+
+    const extracted = extractGoogleQuery(el);
+
+    expect(el).not.toBe(extracted);
+    const $ = cheerio.load(extracted);
```

Comment:
why do we need the cheerio.load on the testing?
i don't think we need this. 
In the testing, we need to hardcode the output, not trying to make it dynamic.
So it will be clearer what we expect and what we assert.

Or maybe @zainfathoni have better knowledge in this.

Reply:
This should be enough:

```js
// what output do you expect from the extracted
expect(extracted).toStrictEqual({});
```

Reply:
The function itself directly generates an HTML string, so we need to actually parse it with `cheerio` to determine the contents.

Reply:
I think we can simplify this issue. Should we also test the HTML contents or just test if the HTML string is different or not?

If the first choice is preferable, I think we can avoid using `cheerio` by using `DOMParser`